### ISSUE TITLE
Option to remove grep symbols from output

### ISSUE THREAD

**Issue by lolbat** [lukehaas/RunJS#363]
Hi folks

I just noticed that the app will use regular expression symbols in the output when present.

Might it be possible to add an option in preferences to disable this? Or perhaps link it to the 'Show Invisibles' preference?

So I get this

```javascript
"itemwep\t'Frost Brand' # nbr 172"
```
and I would prefer this

```javascript
"itemwep   'Frost Brand' # nbr 172"
```

It is super handy when you need something like that though.

**Comment by lukehaas** [comment]
Thanks, I'll look into it.

**Comment by lukehaas** [comment]
@lolbat this is now resolved in the latest version of RunJS - 2.2.2

**Comment by lolbat** [comment]
Thanks muchly.
